Documentation/Design/Classes/Member Variable Declarations.md

```diff
@@ -0,0 +1,189 @@
+# Member Variables
+
+## Feature Overview
+
+Classes may define member variables that each instance of that class contains.
+
+Many languages, such as C# and Java, guarantee class instances are created with space for each member.
+More dynamic languages may declare members without guaranteeing their existence (TypeScript).
+Others such as Python and Ruby forgo declaring them altogether under certain conditions.
+
+
+
+## Commands
+
+### `member variable declare`
+
+`member variable declare` `:` `privacy` `name` `type`
```

Comment:
`privacy` `name` `type` `value`?
You have `member variable declare : protected age int 0` under usage examples below. Unless the 0 is considered part of type or something. I assume value is optional as well.